core/auth/src/main/java/software/amazon/awssdk/auth/credentials/ProfileProviderCredentialsContext.java

```diff
@@ -68,18 +79,33 @@ public static final class Builder {
         private Profile profile;
         private ProfileFile profileFile;
 
+        private Builder() {
+        }
+
+        /**
+         * *
+         * @param profile The profile that should be used to load the configuration necessary to create the credential provider.
+         * @return Returns a reference to this object so that method calls can be chained together.
+         */
         public Builder profile(Profile profile) {
             this.profile = profile;
             return this;
         }
 
+        /**
+         * *
+         * @param profileFile : ProfileFile that has the profile which is used to create the credential provider.
+         *                    This is required to fetch the titles like sso-session defined in profile property*
+         *                    *
```

Comment:
Nit: formatting seems a bit off. 

There's also an extra `*` in the second line for every method